### PR TITLE
lots of doc updates

### PR DESCRIPTION
**Pull request by wmorgan**
* Pull in updates to upgrade doc from 2.11
* Clean up Overview
* Clean up issuer cert rotation

Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

**Comment by wmorgan** [comment]
@alpeb any ideas for how to address the htmltest issue here?

File: linkerd.io/content/2.12/tasks/upgrade.md [hunk 1]
```diff
@@ -1,103 +1,121 @@
...
-In this guide, we'll walk you through how to upgrade Linkerd.
+Read through this guide carefully. Additionally, before starting a specific
+upgrade, please read through the version-specific upgrade notices below, which
+may contain important information about your version.
 
-Before starting, read through the version-specific upgrade notices below, which
-may contain important information you need to be aware of before commencing
-with the upgrade process:
-
-- [Upgrade notice: stable-2.12.0](#upgrade-notice-stable-2-12-0)
```
Comment (alpeb) [inline linkerd.io/content/2.12/tasks/upgrade.md:18]: The link to the stable-2.12.0 notes was lost in your change

**Review by alpeb — COMMENTED** [review]
For the check to be happy I just replaced `../2/tasks/install` with the full url

File: linkerd.io/content/2.12/tasks/upgrade.md [hunk 2]
```diff
@@ -188,87 +218,106 @@ operating correctly, and will list any pods that are still running older
...
+- The linkerd2 Helm chart has been split into two charts: linkerd-crds and
+  linkerd-control-plane.
+- The viz, multicluster, jaeger, and linkerd2-cni Helm charts now rely on a
+  post-install hook required metadata into their namespaces.
 
-The minimum Kubernetes version supported is `v1.21.0`.
+Read on for how to handle these changes as part of the upgrade process.
 
-### Using the CLI
+#### Upgrading a 2.12.0 CLI installation
```
Comment (alpeb) [inline linkerd.io/content/2.12/tasks/upgrade.md:243]: This sounds confusing to me, as we're upgrading _to_ 2.12. Same for the helm header below.

**Review by alpeb — APPROVED** [review]
Thanks, LGTM :+1:

**Pull request merged**